examples/wallets/Cargo.toml

```diff
@@ -0,0 +1,33 @@
+[package]
+name = "examples-wallets"
+publish = false
+
+version.workspace = true
+edition.workspace = true
+rust-version.workspace = true
+authors.workspace = true
+license.workspace = true
+homepage.workspace = true
+repository.workspace = true
+exclude.workspace = true
+
+[dev-dependencies]
+alloy-network = { workspace = true }
+alloy-node-bindings = { workspace = true }
+alloy-primitives = { workspace = true }
+alloy-provider = { workspace = true }
+alloy-rpc-client = { workspace = true }
+alloy-rpc-types = { workspace = true }
+alloy-signer = { workspace = true, features = [
+    "eip712",
+    "mnemonic",
+    "yubihsm",
+] }
+alloy-signer-ledger = { workspace = true }
+alloy-signer-trezor = { workspace = true }
+alloy-sol-types = { workspace = true }
+alloy-transport-http = { workspace = true }
```

Comment:
I think we should be using the `alloy` crates, wdyt @gakonst @mattsse ?

Reply:
Also these should be inline syntax `.workspace = true` since we don't enable anything else